src/test/java/uk/gov/pay/directdebit/mandate/resources/MandateSearchResourceIT.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.pay.directdebit.mandate.resources;
+
+import org.junit.Test;
+
+public class MandateSearchResourceIT {
+    
+    @Test
+    public void searchByInvalidDisplaySize() {
+    }
+
+    @Test
+    public void searchWithMissingGatewayAccountId() {
+    }
+}
```

Comment:
Are these tests coming?

Reply:
no they will be in another PR but i had migrated one of them from the SearchMandateDaoIT. call these placeholder tests that need to be implemented for now. i don't think it's a big deal to have them here.